docs/config.md

```diff
@@ -605,6 +605,12 @@ The following settings are available:
 `executor.memory`
 : The maximum amount of memory made available by the underlying system. Used only by the `local` executor.
 
+`executor.memPerCpu`
```

Comment:
Considering already exist `perJobMemLimit` and `perTaskReserve`, it'd preferable to call this setting `perCpuMemAllocation` for consistency